ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/data/PurchaseInformation.kt

```diff
@@ -88,7 +55,42 @@ internal class PurchaseInformation(
                 true
         },
         managementURL = managementURL,
+        isActive = entitlementInfo?.isActive ?: when (transaction) {
+            is TransactionDetails.Subscription -> transaction.isActive
+            is TransactionDetails.NonSubscription -> true
+        },
```

Comment:
nit: in iOS I've replaced this with `isExpired` that makes more sense for non-subs